config-overrides.js

```diff
@@ -1,3 +1,27 @@
-module.exports = function override(config, env) {
+const { override, getBabelLoader } = require("customize-cra");
+
+module.exports = override((config) => {
+  //@ts-ignore
+  const options = getBabelLoader(config, false).options;
+
+  const originalPreset = options.presets.find((preset) =>
+    preset[0].includes("babel-preset-react-app")
+  );
+  if (originalPreset) {
+    Object.assign(originalPreset[1], {
+      development: true,
+      importSource: "@welldone-software/why-did-you-render",
+    });
```

Comment:
should it be done only in DEV mode?

Reply:
yes, this config is only used with "yarn debug" not "yarn start" or "yarn build".
but I can also wrap this with a "if DEV"

Reply:
ok, great sounds good in that case. Maybe just put a comment on top of the file to explain it